code/Model/Post.php

```diff
@@ -192,6 +192,58 @@ public function fetchByUserId($userId)
         return $models;
     }
 
+    /**
+     * Search string within post title and body and author name.
+     * @param $term
+     *
+     * @return array
+     */
+    public function fetchByTerm($term)
+    {
+
+        /**
```

Comment:
I think this is just a personal preference of mine - I usually like to put any kind of /** comment inside of a method docblock.  Not a big deal either way.  If there's some kind of specific code sniffer we can subscribe to, that might simplify things.